### PR TITLE
Now with correct input/output and N

### PR DESCRIPTION
**Pull request by llogiq**
(At least `make` no longer shows any errors)

### CODE REVIEW

File: src/fasta_redux.rs [hunk 1]
```diff
@@ -0,0 +1,203 @@
+// The Computer Language Benchmarks Game
+// http://benchmarksgame.alioth.debian.org/
+//
+// contributed by the Rust Project Developers
+// contributed by llogiq
+// contributed by teXitoi
```
Comment (TeXitoi) [inline src/fasta_redux.rs]: With a capital T please. Then I'll merge.

**Comment by llogiq** [comment]
Done.

**Comment by TeXitoi** [comment]
Thanks!

**Comment by TeXitoi** [comment]
https://alioth.debian.org/tracker/index.php?func=detail&aid=315138&group_id=100815&atid=413122

**Comment by llogiq** [comment]
Great! :smile:

**Pull request merged**